libraries/AP_CANManager/AP_CANManager.h

```diff
@@ -132,6 +132,11 @@ class AP_CANManager
     private:
         AP_Int8 _driver_number;
         AP_Int32 _bitrate;
+#if HAL_CANFD_SUPPORTED
```

Comment:
header should be the same for with/without CANFD

Reply:
done